src/ImageManipulation.php

```diff
@@ -588,6 +594,20 @@ public function Fill($width, $height)
         });
     }
 
+    /**
+     * Set the quality of the resampled image
+     *
+     * @param int $quality Quality level from 0 - 100
+     * @return AssetContainer
+     */
+    public function Quality($quality)
```

Comment:
Do we want to have some range validation here? If you put a non-sensical value you'll get a `Intervention\Image\Exception\InvalidArgumentException` which is probably good enough. However `castDimension` does explicitly validate the provided dimension.

Reply:
It probably would be worthwhile to add a `@throws` statement.

Reply:
I’ve implemented `castDimension` and added a `@throws` statement